### PR TITLE
build: switch over to temporary fork of goftp.io/server

### PR DESCRIPTION
**Pull request by adamdecaf**
Gitea is moving cloud providers, so our build breaks as goftp.io/server can't be resolved.

Fork: https://github.com/adamdecaf/goftp-server 

Re-write rules: https://golang.org/ref/mod#go-mod-file-replace

### CODE REVIEW

**Review by jmbrown412 — APPROVED** [review]

**Review by atonks2 — APPROVED** [review]

**Comment by jmbrown412** [comment]
ship it

**Review by darwinz — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -5,49 +5,59 @@ go 1.13
...
+	golang.org/x/tools v0.0.0-20201013201025-64a9e34f3752 // indirect
+	gopkg.in/ini.v1 v1.62.0 // indirect
 	gopkg.in/square/go-jose.v2 v2.5.1 // indirect
+	honnef.co/go/tools v0.0.1-2020.1.5 // indirect
 )
+
+replace goftp.io/server v0.4.0 => github.com/adamdecaf/goftp-server v0.4.0
+
+exclude github.com/moov-io/identity v0.2.8
+exclude github.com/moov-io/identity v0.2.9
```
Comment (atonks2) [inline go.mod:63]: But of course, now that you've excluded these versions it's complaining about 0.3.1 🙄 
> ... reading github.com/moov-io/identity/go.mod at revision v0.3.1: unknown revision v0.3.1

Comment (adamdecaf) [reply go.mod:63]: I think we need to get identity out of paygate's dependency tree in order to remove these `exclude` statements.

Comment (atonks2) [reply go.mod]: Sure seems that way. We should probably make sure we don't have private dependencies in other public repos too...

Comment (adamdecaf) [reply go.mod]: Actually. I think it's from the OpenShift image..

Comment (atonks2) [reply go.mod]: Nice find! The build finally passed :D

**Pull request merged**